src/FlowtypeRunner.js

```diff
@@ -0,0 +1,48 @@
+const { exec } = require('child_process');
+const path = require('path');
+const { pass, fail } = require('create-jest-runner');
+
+class FlowtypeRunner {
+  constructor(globalConfig) {
+    this.globalConfig = globalConfig;
+  }
+
+  runTests(tests, watcher, onStart, onResult, onFailure, options) {
+    this.globalConfig = {};
```

Comment:
I don't think you're using this anywhere

Reply:
Nice catch. It was a temporary hack to pass the lint. I changed eslint rules to accept class methods without calling this.